docs/source/examples/Using Interact.ipynb

```diff
@@ -693,6 +693,83 @@
     "interactive_plot"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Interact with multiple functions\n",
+    "\n",
+    "You may want to have a single widget interact with multiple functions. This is possible by simply linking the widget to both functions using the interactive_output() function. The order of execution of the functions will be the order they were linked to the widget.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
```

Comment:
Can you clear the outputs on this notebook?

Reply:
Done, outputs cleared